### ISSUE TITLE
Warnings emitted when trigger is not a pull request

### ISSUE THREAD

**Issue by mh1622** [FranzDiebold/github-env-vars-action#33]
We're using `workflow_dispatch` trigger to run our workflow and therefore receive this warning twice when running franzdiebold/github-env-vars-action@v2.1.0:
```
Warning: Environment variable "GITHUB_HEAD_REF" not set. Cannot set "CI_HEAD_REF_SLUG".
```

This is because `GITHUB_HEAD_REF` is only set when the event trigger is a _pull request_: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions

Perhaps some logic could be implemented to avoid setting `CI_HEAD_REF_SLUG` if `GITHUB_HEAD_REF` is not set?

**Comment by FranzDiebold** [comment]
Hi @mh1622 ,

indeed, `CI_HEAD_REF_SLUG` is not set if `GITHUB_HEAD_REF` is not set. This is what the warning is all about.

Which behaviour would you like to have *exactly*?

**Comment by FranzDiebold** [comment]
Closing because of inactivity.
@mh1622 feel free to reopen!

**Comment by AndrewFarley** [comment]
I was watching quietly-ish to hope this got fixed and now I see it was closed.  All the plugins that generate useless "noise" in the action pipeline view I find very annoying and hard to actually see real errors.  I would prefer this plugin failed gracefully and quietly.  It's useless for me to see on every merge to master this annotation, it doesn't benefit me in any way.  If it can't be disabled by default, can we have a "silent mode" which makes it not output this?  @FranzDiebold

**Comment by FranzDiebold** [comment]
Hi @mh1622 and @AndrewFarley !

Thanks for your feedback!

I just removed the `warning`s in favour of `info` :wink: and released the new version [v2.2.0](https://github.com/FranzDiebold/github-env-vars-action/releases/tag/v2.2.0)!

Happy coding!

**Comment by AndrewFarley** [comment]
Cheers thanks!!!
